modules/ossm-member-roll-create.adoc

```diff
@@ -102,3 +101,41 @@ spec:
 ----
 +
 . Modify the default YAML to add your projects as `members`.  You can add any number of projects, but a project can only belong to *one* `ServiceMeshMemberRoll` resource.
+
+[id="ossm-member-roll-create-member_{context}"]
+== Creating the {ProductName} members
+
+`ServiceMeshMember` resources are created by service mesh users who are not administrators. While project administrators are automatically given permission to create the `ServiceMeshMember` resource in their project, they are not allowed to point it to any `ServiceMeshControlPlane` until the service mesh administrator explicitly grants access to the service mesh. Administrators can modify the mesh users role binding in the control plane project to specify the users and groups that are granted access.
```

Comment:
The last part of the first sentence sounds odd. SMM resources can be created by administrators, too. Maybe the statement should be something like: "... created by service mesh users who don't have privileges to add members to the SMMR directly."

Last sentence: "mesh users" should be `mesh-users`.

Reply:
I think we should change the privileges part to say something like:

Administrators can grant users permissions to access the mesh by granting them the `mesh-user` user role, for example: oc policy add-role-to-user -n <control-plane-namespace> --role-namespace <control-plane-namespace> mesh-user <user-name>